Tensile/Source/client/source/ConvolutionProblem.cpp

```diff
@@ -174,7 +174,7 @@ namespace Tensile
             m_cinPosition  = position++;
             m_coutPosition = position;
         }
-        else if((identifier == "CKYX" and !transposeCK) || (identifier == "KCYX" and transposeCK))
```

Comment:
cppcheck: Expression is always false because 'else if' condition matches previous condition at line 158.
I think this is a false positive. I changed from using 'and' to '&&' and the error seems to have gone away.